packages/pcd/gpc-pcd/src/GPCPCD.ts

```diff
@@ -97,7 +97,15 @@ export type GPCPCDArgs = {
    */
   id?: StringArgument;
   // TODO(POD-P3): Support PODValue of multiple types.
-};
+} /**
+ * POD objects to prove about. Each object is identified by name and prefixed
+ * with {@link PODPCD_ARG_PREFIX} followed by an underscore.  These are not
+ * revealed by default, but a redacted version of their entries will become part
+ * of the claims of the resulting proof PCD, as specified by the proof config.
+ *
+ * See {@link GPCProofConfig} and {@link GPCRevealedClaims} for more
+ * information.
+ */ & Record<PODPCDArgName, PCDArgument<PODPCD>>;
```

Comment:
This is clever.  I woudln't have expected something like this to work, but I assume since you modified consumer-client you've tested it.

I'm not sure I have enough depth of context on TypeScript/Javascript and how the PCD framework is used to know if there could be any gotchas with this approach.  We should get input from @ichub before finalizing this.